test/functional/save-item.js

```diff
@@ -0,0 +1,71 @@
+import { assert } from 'chai';
+import { addAction } from '../util/database';
+import { post } from '../util/mock-client';
+import server from '../../.dist/lib/server';
+import ItemStore from '../../.dist/store/item-store';
+import ModelStore from '../../.dist/store/model-store';
+
+describe('Save Item', () => {
+
+    before(() => {
+        return server.start(8080);
+    });
+
+    after(() => {
```

Comment:
I understand the concern, but I think there's a fair case for group `before` with `after` and `beforeEach` with `afterEach`.

This example is showing them in this order (`before` -> `after` -> `beforeEach`): https://mochajs.org/#hooks